docs/DreamBig/Project Overview (Epic and Sprint Plan)/DreamBig Prototype Epic.md

```diff
@@ -0,0 +1,77 @@
+# DreamBig Prototype epic
+
+## Background / Context 
+
+DreamBig aims to provide a personalised roadmap integrated with Ontrack to support students to
+develop and improve their employability after graduation. In addition, it will provide a platform to
+help set realistic expectations for students graduating into the real world.
+
+## Business Value 
+
+DreamBig will be an integrated platform with Ontrack which helps contributing to student
+employability. This is important to Thoth Tech as we believe in providing educational resources for
+students.
+
+## In Scope 
+
+- User research
+- Design Planning & Documentation
+- Front-end
+- Back-end
+- Database
+- Security
+- Scoping of architecture
+
+## Out of Scope 
+
+- New feature documentation
+- Create User documentation
+
+## What needs to happen 
+
+- A lot of research needs to be done around how this will integrate with the existing platform
+- Design, how will this look?
+- Documentation/Training
+- How students are onboarded
+- Authentication mechanism (Does it use the same login credential as the university one?)
+- Establish QA framework. How will we test?
+- Privacy - how will student privacy be protected?
+- A protype incorporating all design elements and features
+
+## Assumptions / Dependencies 
+
+There is very little done already due to this being a brand-new project
+
+## UX/UI Considerations 
+
+- Keep it simple, consistent layout
+- Similar UI to OnTrack? Brand alignment?
+
+## Analytics Considerations 
+
+How do we want to manage success? What product metrics do we want to measure. For example,
+acquisitions of new users.
+
+## Reg & Compliance Considerations 
+
+Security / privacy considerations with student details What kind of testing will be needed to ensure
+data will be safe from attacks?
+
+## Operations / Support / Training Considerations 
+
+Documentation needs to be created.
+
+## What are the challenges? 
+
+Designing a product that integrate flawlessly with the existing architecture Creating documentation
+
+## Acceptance criteria 
+
+(Informal)
+
+- Validate documentation with managing directors
+- Documentation produced is accurate
+
+
+
+
```

Comment:
**issue:** These empty lines caused the build to fail and do not allow merge. Can you remove them?

Reply:
Ah was wondering why the check was failing, just removed them.